src/components/molecules/card/CardFooter.tsx

```diff
@@ -20,14 +23,30 @@ const useStyles = makeStyles({
     alignItems: 'flex-end',
     justifyContent: 'flex-end',
   },
-});
+  lastDate: {
+    paddingInlineStart: theme.spacing(0.7),
+  },
+}));
 
 const CardFooter: React.FC<IProps> = ({ dateComment }) => {
+  const { t } = useTranslation();
   const classes = useStyles();
-  const dateRange = dateComment?.date_range?.join('-') ?? ''
+  const locale = useLocale();
+  const lastDate = dateComment.last_update == null ? '' : dateFormat(new Date(dateComment.last_update), locale);
+  const dateRange = dateComment.date_range == null ? '' : dateComment.date_range.join('-');
   return (
     <div className={classes.main}>
-      <Typography.Body3>{dateRange}</Typography.Body3>
+      <Typography.Body3>
+        {dateRange}
+        {lastDate && <span>,</span>}
+      </Typography.Body3>
+      {lastDate && (
+        <Typography.Body3>
+          <Box className={classes.lastDate}>
```

Comment:
you have `lastDate` in both `Box` and the inner `span`. one of them is redundant
also, I think you meant to name it `lastUpdateDate` ?

Reply:
@yuvalbl 
ok, I am changing it to the more accurate name. 
about theme.spacing, i think it looks more balanced with less space., this is why I used decimal.
about lastDate class in both box and span, this is because I found in necessary to have space on both sides (see picture attached). let me know what you think. is there a preferred way to insert the needed space? 
 
 with theme.space 1: 
 
![Capture333](https://user-images.githubusercontent.com/88275350/146266565-9b9da8ca-f29b-4418-bd35-7d1b4321c01f.JPG)

with theme space 0.7: 

![Capture555](https://user-images.githubusercontent.com/88275350/146266894-58b2fa45-ec1f-4f4a-a44e-557c0901701f.JPG)